source/js/foundation/pages/youtube-regrets/regrets-reporter-timeline.js

```diff
@@ -0,0 +1,91 @@
+import utility from "../../../utility.js";
+
+let elements = {
+  timeline: `#view-youtube-regrets-reporter .timeline-wrapper .timeline`,
+  segments: `#view-youtube-regrets-reporter .timeline-wrapper .timeline .segment`,
+  button: `#view-youtube-regrets-reporter .timeline-wrapper .btn-timeline-next`,
+};
```

Comment:
It feels like this might be overkill for a piece of UI with only three "parts" to it, but if we're basically going to "build and forget" then this is fine. If we need to work on this in the future, we'll have to let people know that this _will_ require more time due to the fact that it will require rewriting to a cleaner form before we do any more work.

Reply:
@Pomax did you mean this doesn't need to be its own module? Or that we don't need to hold `timeline`, `segments`, and `button` in a `elements` object?

Reply:
the latter - it doesn't offer as much benefit as it did for the youtube regret stories page, which had like 10 elements or something